### PR TITLE
Updated logic to not allow attendee deselection from bill split confirmation page.

### PR DESCRIPTION
**Pull request by PrashantMangukiya**
@Julesssss  pr is ready for review.

### Details

If bill split flow initiated via FAB, we **will Not Allow** attendee deselection on bill split confirmation page. 
If bill split flow initiated from **Group Chat** we **will Allow** attendee deselection on bill split confirmation page.

Discussion https://github.com/Expensify/App/issues/5295#issuecomment-921707123 and https://github.com/Expensify/App/issues/5295#issuecomment-929297039
Confirmation: https://github.com/Expensify/App/issues/5295#issuecomment-929315692

### Fixed Issues
$ https://github.com/Expensify/App/issues/5295

### Tests | QA Steps
**QA Test1:  Bill-split init via FAB**
1.  Click FAB + > Split bill
2. Enter amount and click next
3. It will show attendee select screen, So select two or more attendees and click next
4. You will land at confirmation page. 
So in this case it **Should Not Allow** to deselect attendees on confirmation page.

**QA Test2: Bill-split init via Group Chat**
1. Select Group Chat,
2. Click + button at left of text  box in group chat.
3. Select Split Bill (for Group chat it shows Split Bill. We need to test Split Bill for group chat).
4. Enter amount and click next
5. It will directly land at confirmation page.
So in this case it **Should Allow** to deselect attendees.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
https://user-images.githubusercontent.com/7823358/137534170-7f70be30-e592-436f-b0e4-6bd63b46d319.mov

#### Mobile Web
https://user-images.githubusercontent.com/7823358/137534508-0406df1a-45e0-4d47-8d1d-4bd4c9505109.mov

#### Desktop
https://user-images.githubusercontent.com/7823358/137534730-659b86eb-797c-44a0-9283-e2d1e9b87651.mov

#### iOS
https://user-images.githubusercontent.com/7823358/137534820-68789a49-a162-4880-863d-8e6ec132f3a0.mov

#### Android
https://user-images.githubusercontent.com/7823358/137534934-01af29a5-4162-477a-af19-66aff6f5a869.mov

### CODE REVIEW

**Review by parasharrajat — DISMISSED** [review]
LGTM & tests well.

cc: @Julesssss ready for final review and merge when n6-hold is lifted.

File: src/components/IOUConfirmationList.js [hunk 1]
```diff
@@ -65,6 +65,9 @@ const propTypes = {
         phoneNumber: PropTypes.string,
     })).isRequired,
 
+    /** Amount split belongs to group or not */
```
[the remark this answers is not in the document]
Comment (PrashantMangukiya) [reply src/components/IOUConfirmationList.js:68]: @parasharrajat thanks, I will update instruction as per your suggestion asap.

[the remark this answers is not in the document]
Comment (PrashantMangukiya) [reply src/components/IOUConfirmationList.js]: Changes done. thanks.

**Review by parasharrajat — APPROVED** [review]
LGTM. Tests well.

Ready for final review and merge when n6 is lifted. cc: @Julesssss

**Review by Julesssss — APPROVED** [review]

**Pull request merged**